packages/plugin-auto-tracking-browser/src/auto-tracking-plugin.ts

```diff
@@ -62,11 +105,16 @@ export const autoTrackingPlugin = (options: Options = {}): BrowserEnrichmentPlug
           return false;
       }
     }
-    const tag = element.tagName.toLowerCase();
+    /* istanbul ignore next */
+    const tag = element?.tagName?.toLowerCase?.();
+
+    // We only track these limited tags, as general text tag is not interactive and might contain sensitive information.
     /* istanbul ignore if */
-    if (!tagAllowlist.includes(tag)) {
+    if (!DEFAULT_TAG_ALLOWLIST.includes(tag)) {
```

Comment:
Won't this prevent users from selecting other elements? Should we remove this and only use cssSelectorAllowList?

Reply:
Explained in https://github.com/amplitude/Amplitude-TypeScript/pull/589#discussion_r1349205879.